src/amo/components/InstallWarning/index.js

```diff
@@ -67,7 +67,7 @@ export class InstallWarningBase extends React.Component<InternalProps> {
           isFirefox({ userAgentInfo }) &&
           clientApp === CLIENT_APP_FIREFOX &&
           addon.type === ADDON_TYPE_EXTENSION &&
```

Comment:
both the clientApp and addon.type restrictions don't exist on the api | addons-server so does it make sense to have them on addons-frontend?   Why would we _want_ to have these extra restrictions?

Reply:
These are part of the requirements that existed for Recommended add-ons. Ad add-on was only going to be recognized as recommended for badge/warning purposes if it was an extension, and also never on the `android` site (`clientApp === CLIENT_APP_ANDROID`). These requirements have not changed for Recommended add-ons, so the code remains.

Reply:
> These requirements have not changed for Recommended add-ons, so the code remains.

We're not badging for Android?  Are you differentiating "Android" and "Fenix" here?

Reply:
I am not. When I first asked @jvillalobos about how to handle this he indicated that we should just keep it as is for now, as I believe we are still figuring out exactly what we want to do about Fenix. I have asked some questions related to this here: https://github.com/mozilla/addons-frontend/issues/9606#issue-690262837

Reply:
Also note that this code is specifically about people visiting the `android` site, and has nothing to do with user agent sniffing.